stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.19.1
+version: 1.20.1
```

Comment:
This should be bumped to `1.21.0` now since another PR was merged earlier today.